Wikipedia/Code/Session.swift

```diff
@@ -1,7 +1,7 @@
 import Foundation
 
 public extension URLRequest.CachePolicy {
-    static let noReallyForceRemoteOnly: UInt = 99
+    static let noPersistentCacheOnError: UInt = 99
```

Comment:
Would it help to have our own `CachePolicy` enum that has a case to wrap an underlying `URLRequest.CachePolicy`?
```
enum CachePolicy {
    case foundation(URLRequest.CachePolicy)
    case noPersistentCacheOnError
}
```

Reply:
@joewalsh agreed, that feels much cleaner. This is done!